server/api/auth/recover.js

```diff
@@ -41,7 +41,7 @@ module.exports = {
     await cache.login.makeLogin({ id: verifyUuid })
     const verifyToken = await auth.token.getToken(auth.token.tokenKinds.verify, {
       verifyId: verifyUuid,
-      register: false,
+      kind: 'recover',
```

Comment:
Should we encode the kind as a typescript enum?

Reply:
Ideally the parameter to getToken should be a single discriminated union so we also have the contents of the token defined by an interface, however I’m not sure how compatible that is with pure JS code so that might need to wait for when we fully convert auth to TS